apps/condo/domains/common/utils/helpers.ts

```diff
@@ -1,7 +1,23 @@
+import { ParsedUrlQuery } from 'querystring'
+import get from 'lodash/get'
+
 export const formatPhone = (phone?: string): string => {
     if (!phone) {
         return phone
     }
 
     return phone.replace(/(\d)(\d{3})(\d{3})(\d{2})(\d{2})/, '$1 ($2) $3-$4-$5')
-}
\ No newline at end of file
+}
+
+export const getFiltersFromQuery = <T>(query: ParsedUrlQuery): T | Record<string, never> => {
+    const filters = get(query, 'filters')
```

Comment:
Can be reduced to `query.filters`, because `get` from `lodash` is not utilising `defaultValue` argument.